docker-compose.yml

```diff
@@ -6,7 +6,56 @@ volumes:
 services:
 
   # Safe Transaction Service
+  hardhat:
+    image: ethereumoptimism/hardhat:latest
```

Comment:
Why do we requite this? This should only be required for local testing, I was assuming we should create an example that works with external rpc nodes.

Reply:
Ah my understanding was that we should go for a local setup first as suggested here https://github.com/gnosis/safe-client-gateway/issues/564#issuecomment-909318311

Reply:
> Why do we require this?

This is not final, but the simplest way I could get a container running a hardhat node

Reply:
I have removed hardhat for now in favour of an `RPC_NODE_URL` env var